src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -476,6 +476,14 @@ public void testListZeroTenants() throws Exception {
     checkRequestHeaders(interceptor);
   }
 
+  @Test
+  public void testDeleteTenant() throws Exception {
+    TestResponseInterceptor interceptor = initializeAppForUserManagement("{}");
+    // should not throw
+    FirebaseAuth.getInstance().getTenantManager().deleteTenantAsync("TENANT_1").get();
+    checkRequestHeaders(interceptor);
```

Comment:
Let's also assert the URL of the outgoing request and make sure `TENANT_1` is being included in the request.

Reply:
Done. How do I assert that that it was a DELETE request?

Reply:
Something like the following should work:

```
interceptor.getResponse().getRequest().getMethod()
```

Reply:
Done.